cloud/services/create-a-pg-service.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Create a PostgreSQL service
+excerpt: Create a PostgreSQL service in Timescale Cloud
+product: cloud
+keywords: [services, postgresql, create, install]
+---
+
+# Create a PostgreSQL service
+
+Timescale Cloud is a hosted, cloud-native TimescaleDB service that allows you to
+quickly spin up new TimescaleDB instances. In addition to your TimescaleDB
+databases, you can also create services that run PostgreSQL, if you need to
```

Comment:
Should we qualify with 'regular PostgreSQL' here? Just to really drive home the message that 'TimescaleDB _is_ PostgreSQL'?

Reply:
Yeah, I was wondering that myself, but couldn't come up with an appropriate adjective. I think we use 'vanilla' elsewhere, but it feels like it might be US-centric?

Reply:
How about 'standard PostgreSQL'?

Reply:
Yeah, that could work. We'll need to update the style guide, too.

Reply:
Hrm. I might be belabouring the point now 🤔

Reply:
I think it's good!